schema.sql

```diff
@@ -4,5 +4,8 @@ CREATE TABLE animals(
 	date_of_birth DATETIME,
 	escape_attempts INT,
 	neutered BOOLEAN,
-	weight_kg DECIMAL
+	weight_kg DECIMAL,
 );
+
+ALTER TABLE animals
+vet_clinic-# ADD COLUMN species VARCHAR(50);
```

Comment:
- [ ] [OPTIONAL] Great job adding the `ALTER TABLE` as the preview reviewer suggests, however kindly consider removing `vet_clinic-# ` from the start of the line (line `11`). 
     #### ⚠️ Please try to do this before merging the PR

Reply:
Hi @Ismailco!

Great, that part will be deleted before doing the merge. Thanks for the recomendation!